help/destinations/catalog/advertising/snap-inc.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Snap Inc
+description: Use your audience segments in the Snapchat Ads Platform.
+---
+
+# Snap Inc
+
+## Overview {#overview}
+
+[Snapchat Ads](https://forbusiness.snapchat.com/) are made for every business, no matter the size or industry. Become a part of Snapchatters' everyday conversations with full-screen, digital ads that inspire action from the people that matter most to your business.
+
+>[!IMPORTANT]
+>
+>This documentation page was created by the *Snap Inc* team. For any inquiries or update requests, please contact them directly at *dev-support@snap.com*
+
+## Use cases {#use-cases}
+
+This destination allows marketers to import user segments that they built using the Adobe platform into Snapchat Ads and use them to target their ads. 
+
+## Prerequisites {#prerequisites}
+
+To use this destination, you must have a Snapchat Ads account. Please refer to this documentation for information about how to create one:
+
+[Get Started with Snapchat Advertising](https://businesshelp.snapchat.com/s/article/overview?language=en_US)
+
+## Limitations {#limitations}
+
+* Snap Inc does not support multiple identities for a given Audience Segment. Please map only one identity when activating a segment.
+* Snap Inc does not support renaming segments. To rename a segment, you must deactivate, rename it, and then activate it.
+* It is not possible to define a retention period for an audience segment's members. All members have lifetime retention and will be in the segment until they are removed..
+
+
+## Supported identities {#supported-identities}
+
+The *Snap Inc* destination supports the activation of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+All identifiers sent to the *Snap Inc* destination must be hashed in SHA-256 format. To hash plain text identifiers before sending them to the destination, check the [!UICONTROL Apply transformation] option when mapping target identifiers for the destination. 
+
+**Unhashed identifiers will not be accepted by the Snap Inc destination and sending them could cause errors.**
```

Comment:
we could call this out even more visibly by using a WARNING note, like this: 
```
>[!WARNING]
> 
> Unhashed identifiers will not be accepted by the Snap Inc destination and sending them could cause errors.
```

warning notes render in a red note block, so they're very hard to miss in the documentation.